models/marts/web_events/intermediate/snowplow_sessions__gclids.sql

```diff
@@ -0,0 +1,42 @@
+with sessions as (
+
+    select * from {{ref('snowplow_sessions')}}
```

Comment:
this model needs a test

Reply:
a visitor can have more than one `session_index` so the `user_snowplow_domain_id` is going to fan out according to how many session "indices" they have...

Reply:
lets put a test on this for surrogate key: user_snowplow_domain_id, session_index